### DISCUSSION TITLE
Should we skip mypy testing at the initlal stage of development?

### DISCUSSION THREAD

**Discussion by lwaekfjlk** [ulab-uiuc/research-town#5] [Ideas]
At the initial stage of development, mypy testing might take a lot of time to do. Should we keep that? @Kunlun-Zhu @Monstertail @chengzr01

**Comment by Monstertail** [comment]
Maybe we could refer to SGLang(https://github.com/sgl-project/sglang) at the very beginning with no mypy testing.

Refer to workflow of vLLM later.

**Reply by lwaekfjlk** [reply]
Nice~ That is a good example. @Monstertail please help contribute to python-project-template repo and transfer them to here.
